src/components/OnboardingForm.tsx

```diff
@@ -126,12 +122,37 @@ export const OnboardingForm = () => {
     setFieldErrors(errors);
 
     if (Object.keys(errors).length === 0) {
-      setFormIsValid(true);
+      return true;
+    }
+
+    if (isFromSubmitted) {
+      if (errors.email) {
+        var email: HTMLElement | void = document
+          .getElementById("email")
+          ?.focus();
+        return false;
+      }
+      if (errors.orgName) {
+        var orgName: HTMLElement | void = document
+          .getElementById("org-name")
+          ?.focus();
+        return false;
```

Comment:
I think you don't need to catch this variable, Even you need it, it's better to use let

Reply:
Thank you for mentioning it.